docs/pages/careers/staff-engineer-pigment-css.md

```diff
@@ -0,0 +1,126 @@
+- **MUI has a large codebase**. You may bang your head against the wall at times, and then write tests to assist future you 😌.
+  The work you will be doing is somewhat unique and idiosyncratic. You probably have not had a similar role before.
+- **In open-source, you're faced with a nonstop stream of bug reports and support requests**. That means you need to develop an intuition for when to ignore something, and when to dig in further.
+
+## Who we're looking for
+
+We're looking for someone with strong core JavaScript skills. More important than specific technical skills though is that you're a strong problem solver who loves to learn. Details matter to you.
+
+### Required
+
+- **Expertise in the modern JavaScript ecosystem**.
+  MUI's open-source projects are built on the shoulders of giants, making use of technologies such as ES2021, TypeScript, Node.js, React, Next.js, Webpack, and Babel.
+- **A track record of demonstrating an eye for product and solving real-world user problems**. If you have a knack for solving problems at the root cause, shipping beautiful user interfaces and intuitive APIs, we want you on our team.
+- **Experience building and shipping production code in a team setting** with a passion for writing tested, performant, and high-quality code.
+- **Strong written and verbal communication skills**.
+  As part of the team, you'll interface both directly and indirectly with community members and enterprise customers, and contribute to user documentation. Clear communication is fundamental in creating intuitive and compelling resources.
+- **Ability to dive into complex problems**.
+  You should be able to quickly assess, understand, and iterate upon aspects of our codebase.
+- **Ready and willing to ask and answer questions**.
+  If you're comfortable saying you're unsure, asking for help; but equally reaching out to assist others, you'll be an incredible addition to our team. We thrive because of continuous learning. First-time mistakes should be celebrated, not blamed.
+- **Avoid monolithic deliverables**.
+  You scope and stage your work into well-defined milestones to ship.
+- **Past work with front-end infrastructure**.
+  Perhaps you've written a Babel plugin, or even worked on a problem similar to the ones Pigment CSS aims to address.
+  It would be great if you could address this in your cover letter!
+
+### Nice to have (but not required)
+
+- **You've maintained an active repository before**.
+  Maybe you've helped maintain a popular open-source repository, or perhaps you've worked on internal repositories that saw contributions from multiple teams.
+  Previous experience with highly active repository workflows is a definite plus for this role.
+- **You have contributed code to some CSS-in-JS library**. A history of contributing to some CSS-in-JS library is a plus.
```

Comment:
This should come first, imo

Reply:
It feels like specific knowledge can be gained a LOT faster than product engineering and technical skills. I'm saying this because I have many people who had one successful project have multiple ones on unreleased product engineering spaces. So to me, the point of this line is to have people who demonstrate that they cared about the problem space (we don't care about the knowledge they gained, no much).

So the order between the two depends on what we want to prioritize between:

- high product engineering skills
- low drive to solve this problem

vs.

- low product engineering skills
- high drive to solve this problem

I think the former is better, someone applying is already a signal that he cares about this problem.

Reply:
I agree that this is very specific. CSS-in-JS itself is a solved problem and we can just re-use those. Everything else is covered through `Expertise in the modern JavaScript ecosystem`.
This leaves us with experience writing Babel/Webpack/Rollup plugins or any other equivalent work with bundlers.